wordpress-mobile/WordPress-iOS
# Flip icons for RTL layout

Fixes #9478 

This PR fixes some issues with RTL layout:
- Flipping icons.
- Flipping button insets.
- Flipping layout alignment.

**To Test:**
Please check every screenshot and check that it looks like the left one on LTR and like the right one on RTL. The middle one is how it used to look before this fix.

To change the app to RTL layout you can go to `Edit scheme` and then under `Run -> Options -> Application Language`, choose : `Right-to-Left Pseudolanguage`.

### Reader detail
- [x] Title alignment.
- [x] Bottom comment icon flipped.
- [x] Bottom Save button insets.

![artboard 2](https://user-images.githubusercontent.com/9772967/41015906-f854c2e0-6911-11e8-9cb4-c7634ed9a8c2.png)

### Activity
- [x] Post icon flipped.

![artboard 3](https://user-images.githubusercontent.com/9772967/41015907-f898cb02-6911-11e8-9c52-c2f36e5aa4d6.png)

### My Sites
- [x] Blog posts icon flipped.
- [x] Comments icon flipped.
- [x] Menus icon flipped.

![artboard 4](https://user-images.githubusercontent.com/9772967/41015908-f8c022ec-6911-11e8-8b44-742b2d858eeb.png)

### Blog posts
- [x] Title alignment.
- [x] bottom view icon.
- [x] bottom edit icon.

![artboard](https://user-images.githubusercontent.com/9772967/41015909-f8ead906-6911-11e8-8f8b-2891e0c79032.png)

### Aztec editor
- [x] Unordered list (With the dots)
- [x] Ordered list icon (with the numbers)

![artboard](https://user-images.githubusercontent.com/9772967/41560823-4043c67e-7316-11e8-9d42-29ebb4434efa.png)

@ctarda Could you please take a look?
@malinajirka Could you please check that I didn't flip something we should not flip? 😆
cc @iamthomasbishop
